connect-inject/handler.go

```diff
@@ -8,7 +8,7 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/deckarep/golang-set"
+	mapset "github.com/deckarep/golang-set"
```

Comment:
I'm assuming your IDE brought this in, happens to everyone, could you remove it?